src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -240,6 +241,7 @@ class BtreeLBAManager : public LBAManager {
       len,
       intermediate_key,
       actual_addr,
+      0,
```

Comment:
> Similar to remapping, for partial cloning, we should also either disable it or read the buffer out in order to calculate the checksum, right?

Um, not really, because partial cloning doesn't touch the actual data's lba mapping, so the data doesn't need to be read out from disk.

> 
> Worth a TODO.

Reply:
I see.

How about adding the comment instead:
```
// crc will only be used and checked with LBA direct mappings
// also see pin_to_extent(_by_type)
```